composer.json

```diff
@@ -13,13 +13,14 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
+        "php": "^7.1",
         "psr/http-message": "^1.0",
-        "paragonie/random_compat": "^1.1|^2.0|^9.99"
+        "psr/http-server-middleware": "^1.0",
+        "slim/slim": "^4.0"
     },
     "require-dev": {
-        "slim/slim": "~3.0",
-        "phpunit/phpunit": "^4.0"
+        "phpunit/phpunit": "^7.5",
+        "slim/psr7": "^0.5.0"
```

Comment:
We don't need `Slim/psr7` as a dependency. We can use prophecies to write the unit tests.